front/components/assistant/conversation/ConversationViewer.tsx

```diff
@@ -234,20 +252,33 @@ export default function ConversationViewer({
         eventIds.current.push(eventPayload.eventId);
         switch (event.type) {
           case "user_message_new":
-            // Ignore user_message_new event as the optimistic mechanism covers it
-            break;
```

Comment:
Removing this, fixes multi-streams in case two users are on the same conversation at the same time.